sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -40,6 +40,16 @@
      * In case of duplicate keys within the scopes, only the first entry is
      retained, while all subsequent duplicate entries are discarded.
 
+* Resolved an issue where activity tags of various object types, including
+  double, float, and others, were previously formatted using
+  `CultureInfo.CurrentCulture`. This behavior caused inconsistencies in tag
+  value formatting depending on the regional settings of the machine where the
+  application was running. Such inconsistencies could lead to challenges in data
+  analysis and cause test failures in environments with differing cultural
+  settings. The fix ensures uniform and culture-independent formatting of
+  activity tag values, aligning with consistent data representation.
+  ([#39470](https://github.com/Azure/azure-sdk-for-net/pull/39470))
```

Comment:
```suggestion
  ([#40000](https://github.com/Azure/azure-sdk-for-net/pull/40000))
```

Reply:
I need to fix an URL to issue rather than PR. 39470 is the reported issue number.

Reply:
All the other links in this Changelog link to the PR instead of the Issue.

Reply:
We did not create issue for those. If you scroll down, might find some entry pointing to an issue.